src/main.rs

```diff
@@ -35,35 +28,27 @@ fn env_var(n: &str) -> String {
 }
 
 // overloaded function. no body is treated as a get, body is treated as a put
-fn cloudflare_api(client: &hyper::client::Client, url: &str, body: Option<&str>) -> Result<Value, String> {
+fn cloudflare_api(url: &str, body: Option<String>) -> Result<Value, String> {
```

Comment:
Any reason not to still allow the client to be reused?

Reply:
I'm not sure why I did this, possibly because happened because I removed `hyper` first, then tried a couple of options while getting things to work.

In any case, I just pushed a commit that restores client sharing.